File: ballast-core/npm/src/application/models/game-dto.ts
import { IBoardDto } from "./board-dto";
import { IPlayerDto } from "./player-dto";
import { IVesselDto } from "./vessel-dto";

export interface IGameDto {
    id: string;
    board: IBoardDto;
    vessels: IVesselDto[];
    players: IPlayerDto[];
    createdOnDateIsoString: string;
    startedOnDateIsoString: string | null;
    endedOnDateIsoString: string | null;
}

File: ballast-core/dotnet/src/application/models/GameDto.cs
using System;

namespace Ballast.Core.Application.Models
{
    public class GameDto
    {
        public Guid Id { get; set; }
        public BoardDto Board { get; set; }
        public VesselDto[] Vessels { get; set; }
        public PlayerDto[] Players { get; set; }
        public string CreatedOnDateIsoString { get; set; }
        public string StartedOnDateIsoString { get; set; }
        public string EndedOnDateIsoString { get; set; }
    }
}